src/MultiMapManager/MultiMapManager.js

```diff
@@ -72,7 +72,7 @@ class MultiMapManager extends React.Component {
       const children = !Array.isArray(rawChildren) ? [rawChildren] : rawChildren
 
       return children.map((child, i) => {
-        if (child.type?.displayName === '_Map' || child.type?.toString?.().startsWith('function _Map')) { // check to see if `child` is a Map component, this isn't the best check ever but it works
+        if (child.props.isMultiMap) {
```

Comment:
🎉

Reply:
@PatchesMaps just a note- this will be an explicit prop required by IAs `isMultiMap`...I updated the docs to reflect that as well